domain/src/main/java/com/teamagam/gimelgimel/domain/map/SetVectorLayerVisibilityInteractor.java

```diff
@@ -34,8 +34,16 @@ public SetVectorLayerVisibilityInteractor(@Provided ThreadExecutor threadExecuto
     protected Iterable<SubscriptionRequest> buildSubscriptionRequests(
             DataSubscriptionRequest.SubscriptionRequestFactory factory) {
         SubscriptionRequest setVisibilityRequest = factory.create(
-                Observable.just(new VectorLayerVisibilityChange(mVectorLayerId, mIsVisible))
-                        .doOnNext(mVectorLayersVisibilityRepository::changeVectorLayerVisibility));
+                Observable.just(null),
```

Comment:
Resolved this is passing the VectorLayerVisibilityChange object instead of id+visibilityFlag